packages/frontend/src/pages/scaling-tvl/view/ScalingTvlView.tsx

```diff
@@ -54,9 +54,7 @@ export function ScalingTvlView({
       alignCenter: true,
       minimalWidth: true,
       headClassName: 'md:pl-4',
-      getValue: (entry, index) => (
-        <IndexCell entry={entry} className="md:pl-4" index={index + 1} />
-      ),
+      getValue: (entry) => <IndexCell entry={entry} className="md:pl-4" />,
```

Comment:
Ok, nice numbers, but I think we do want to have the initial numbers, in case that js loads slowly, we minimise the content shift

Reply:
+1